merlin/schema/schema.py

```diff
@@ -53,8 +53,6 @@ def __post_init__(self):
                 dtype = np.dtype(self.dtype.numpy_dtype)
             elif hasattr(self.dtype, "_categories"):
                 dtype = self.dtype._categories.dtype
-            elif hasattr(self.dtype, "kind"):
-                dtype = np.dtype(self.dtype.kind)
```

Comment:
I think we still need to handle string dtypes (the parquet change only handles int/float nullable dtypes):

```python
            elif isinstance(self.dtype, pd.StringDtype):
                dtype = np.dtype("O")
```

Reply:
To clarify - We *could* add a check for pandas string types in the parquet reader, but I believe they are created in other places anyway...

Reply:
We avoid creating nullable integer and float dtypes in parquet now, but we weren't creating `StringDtype` there anyway (I think - I need to check that).

Reply:
Okay - I think you are right. I missed that some categorical columns are producing StringDtype - I'll try another tweak to the parquet reader and see if we can just drop the block as you are suggesting here.

Reply:
Okay - It does seem that your original change here works fine if #66 is also merged :)